core/state_transition.go

```diff
@@ -504,19 +504,25 @@ func (st *StateTransition) innerTransitionDb() (*ExecutionResult, error) {
 		// are 0. This avoids a negative effectiveTip being applied to
 		// the coinbase when simulating calls.
 	} else if kromaConfig != nil {
-		gasUsed := new(big.Int).SetUint64(st.gasUsed())
-		feeDist := st.evm.Context.FeeDistributionFunc(blockNum, gasUsed, st.evm.Context.BaseFee, effectiveTip)
-		st.state.AddBalance(params.KromaValidatorRewardVault, feeDist.Reward)
-		st.state.AddBalance(params.KromaProtocolVault, feeDist.Protocol)
+		if st.evm.ChainConfig().IsKromaMPT(st.evm.Context.Time) {
+			effectiveFee := new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), effectiveTip)
+			st.state.AddBalance(st.evm.Context.Coinbase, effectiveFee)
```

Comment:
How can we set Coinbase (miner) to KromaProtocolVault address? Currently `miner` is zero address for all blocks

Reply:
The SuggestedFeeRecipient value sent by the CL as a payload will become the Coinbase.